test/http/nominatim.yml

```diff
@@ -1,12 +1,12 @@
-/search?accept-language=&format=xml&q=Hoddesdon&viewbox=-0.559%2C51.766%2C0.836%2C51.217:
+/search?accept-language=&extratags=1&format=xml&q=Hoddesdon&viewbox=-0.559,51.766,0.836,51.217:
```

Comment:
No, there's some slight differences involving the order of url parameters, and their encoding, and how toleratant webmock was being about matching regexps against their various permutations. I didn't think it was all that important (since it's just refactoring the tests, not the code) so I changed the fixtures to work rather than ratholing.